pkgs/development/libraries/smooth/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+
+, gtk
```

Comment:
Can we write gtk2 or gtk3 here or is it important we use the one from gnome?

Reply:
@SCOTT-HAMILTON can you answer this and fix the merge conflict?

Reply:
Fixed, I'm using gtk3.